ci/upstream-dev-environment.yml

```diff
@@ -1,17 +1,12 @@
-name: uxarray_build
+name: uxarray_upstream_build
 channels:
   - conda-forge
 dependencies:
   - netcdf4
   - numba
-  # pinning the numpy version for numba compatibility
-  - numpy
+  - numpy>=1.2, <1.25
```

Comment:
Is this version pinning needed? I think it was resolved a few releases ago for the regular CI.

Reply:
Oh good point! It was something I was trying along the way. Let me revert this and see if everything still holds